### ISSUE TITLE
Please combine WU API calls

### ISSUE THREAD

**Issue by taogeek** [squix78/esp8266-weather-station-color#6]
The 3 API calls can be combined into one by catenating the requests into one URL like this  /conditions/forecast10day/astronomy/

**Comment by G6EJD** [comment]
Yes but this increases the processing complexity and separating out the three responses, does it really make it any more efficient?

**Comment by taogeek** [comment]
I just did it as a test, and yes, I think it's worthwhile in this case.  

There is no change to processing the results.  All code stays the same.  But instead of 3 calls for data, it's 1.  And it's two lines in the WU code -- easy enough that I could do it!

**Comment by squix78** [comment]
Thank you very much for this suggestion. I decided to go another way and actually split the different API endpoints in separate classes due to memory problems and for cleaning the code.  Hence I will close this issue as "Won't fix". I hope you agree

**Comment by pilnikov** [comment]
astronomy and forecast prognosys request can do once time per day. Only condition need update after 15-30 minutes. 
`updateData(bool conditions, bool astronomy, bool forecast);`
And a call with conditions

**Comment by G6EJD** [comment]
the forecast data is updated at regular intervals throughout the day, but updating astronomy only once/day is a good idea.
